src/main/java/data/Recipe.java

```diff
@@ -26,32 +28,99 @@
 
   private String name;
   private String description;
+  private Set<String> tags;
+  private Set<String> ingredients;
   private List<Step> steps;
-  private List<SpinOff> spinOffs;
+  private Set<SpinOff> spinOffs;
```

Comment:
Are you sure this needs to be a field on the Recipe? When will you need to access the spinoffs of a recipe from that recipe?

Reply:
I thought we would have a feature where users can see how other people have spun off a recipe, so this is how I was planning on storing that feature.

Reply:
Got it, sounds good to me then.